frontend/workflows/audit/src/logs/index.tsx

```diff
@@ -97,13 +97,18 @@ const AuditLog: React.FC<AuditLogProps> = ({ heading, detailsPathPrefix, downloa
             onQuickSelect={(start, end) => {
               setStartTime(start);
               setEndTime(end);
-              setTimeRangeKey(`${startTime}-${endTime}`);
+              setTimeRangeKey(`${startTime}-${endTime}-${new Date().toString()}`);
```

Comment:
Should we break this out into it's own function that way you're not having to update 3 places?